maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,109 @@
+            fn_call = output.fnc_call
+            tool_call = tool_ctx.function_tools.get(fn_call.name)
+            tool_call = trace.tool_call(
+                {
+                    "id": fn_call.id,
+                    "name": fn_call.name,
+                    "description": tool_call.__livekit_tool_info.description
+                    if tool_call.__livekit_tool_info.description is not None
+                    else "",
+                    "args": fn_call.args,
+                    "start_timestamp": start_timestamp,
+                }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)